gradlew.bat

```diff
@@ -1,92 +1,92 @@
-@rem

```

Comment:
Probably a good idea to fix these line endings but I assume that doesn't have anything to do with this issue. If so, please do that in a separate PR (or split it into a separate commit and squash the other two!)